src/client/graphql/fragments/CampaignDataFragment.graphql

```diff
@@ -15,6 +15,7 @@ fragment CampaignData on Campaign {
     }
   }
   code
+  # TODO: Remove owner, since it's deprecated
```

Comment:
[Code Scout] : Maybe it would be nice to go the extra mile here and actually remove it?

Reply:
Maybe. Just thought that this change might require work in other places as well, since I assume we use this data somewhere. And didn't want to make this PR even bigger.

Reply:
I took the opportunity and I've created a [ticket](https://hedvig.atlassian.net/browse/GRW-768) for it so we can work on it outside this PR.